lib/sms.rb

```diff
@@ -0,0 +1,15 @@
+require 'twilio-ruby'
+
+class SMS
+account_sid = ENV['ACCOUNT_SID']
+aut_token = ENV['AUTH_TOKEN']
+
+def initialize(@client = Twilio::REST:Client.new(account_sid, auth_token))
```

Comment:
formal argument cannot be an instance variable
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)<br>unexpected token tRPAREN
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)